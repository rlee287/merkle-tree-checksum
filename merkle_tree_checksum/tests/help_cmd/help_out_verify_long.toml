bin.name = "merkle_tree_checksum"
args = "verify-hash --help"
status.code = 1
