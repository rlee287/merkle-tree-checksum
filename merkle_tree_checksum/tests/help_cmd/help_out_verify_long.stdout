Verify Merkle tree hashes

Usage: merkle_tree_checksum verify-hash [OPTIONS] <FILE>

Arguments:
  <FILE>
          File containing the hashes to check

Options:
      --fail-fast
          Skip checking the rest of the files when a hash mismatch is detected.

  -h, --help
          Print help (see a summary with '-h')
